application/pom.xml

```diff
@@ -16,6 +16,10 @@
             <groupId>org.carlfx</groupId>
             <artifactId>cognitive</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.pixelduke</groupId>
+            <artifactId>fxcomponents</artifactId>
+        </dependency>
```

Comment:
Can these be in **test** scope for now?

Reply:
No, the custom control is only needed in the Sampler, good point! 
Done!

Reply:
> Can these be in **test** scope for now? Or does the custom control need them?

No, the custom control is only needed in the Sampler, good point!
Done!